react-native-reanimated.d.ts

```diff
@@ -371,6 +392,29 @@ declare module 'react-native-reanimated' {
       config: DecayConfig,
     ): BackwardCompatibleWrapper;
 
+    // reanimated2 animations
+    export function withTiming(
+      toValue: number,
+      userConfig?: Omit<TimingConfig, 'toValue'>,
+      callback?: (isCancelled: boolean) => void,
+    ): number;
+    export function withSpring(
+      toValue: number,
+      userConfig?: Omit<SpringConfig, 'toValue'>,
+      callback?: (isCancelled: boolean) => void,
+    ): number;
+    export function cancelAnimation<T extends SharedValue>(
```

Comment:
Also, this generic type 'SharedValue' requires 1 type argument.

Reply:
try to use SharedValue<any> here